sdk/appconfiguration/azure-appconfiguration/tests/test_azure_appconfiguration_client.py

```diff
@@ -109,43 +106,6 @@ def test_set_existing_configuration_setting_label_wrong_etag(self, appconfigurat
         with pytest.raises(ResourceModifiedError):
             client.set_configuration_setting(to_set_kv, match_condition=MatchConditions.IfNotModified)
 
-    @app_config_decorator
-    @recorded_by_proxy
-    def test_set_configuration_setting_etag(self, appconfiguration_connection_string):
```

Comment:
Why delete them?

Reply:
I did a little refactor and renaming:
- with etag test moved to `test_get_configuration_setting_with_etag`(line153)
- no etag test is duplicate with existing test `test_set_existing_configuration_setting_label_etag`